### ISSUE TITLE
Improove --help of the cli

### ISSUE THREAD

**Issue by PeterStolz** [gaogaotiantian/viztracer#193]
I like viztracer, but the help promp could be greatly improoved.

When running the latest version of viztracer the coommand line output looks like:
```bash
#viztracer -h
usage: viztracer [-h] [--rcfile [RCFILE]]

optional arguments:
  -h, --help         show this help message and exit
  --rcfile [RCFILE]
```   
In my opinion the same help page should be printed when `-h` or `--help` are present, as it is printed when running viztracer without an argument.   
The `--rcfile` option is also described in the extended help page in there so I don't see a reason for this behavior.

**Comment by gaogaotiantian** [comment]
You are right, this is a bug I introduced when I added the pre-parser for rcfile. Your fix seems to be working, but in case of this kind of problem happening in the future, could you add some regression test? You can refer to ``tests/test_cmdline.py``, in ``test_no_file`` I checked ``help`` in the output string. Obviously that was not enough. You can add a ``test_help`` method in that class which confirms that ``viztracer``, ``viztracer -h`` and ``viztracer --help`` all produce useful information. The test cause should not be hard to follow. However, you can always ask me if you have more questions. Fixing a bug is good, but making sure it won't happen again is even better.

However, I'd understand if you don't think that's something you'd want to do. Then I can accept your PR and do the test part myself. Feel free to let me know!

Thank you for your contribution.

**Comment by PeterStolz** [comment]
Hi, I added the testcase in the PR. Let me know if they should be improved.
